### ISSUE TITLE
Buying Station only offers 1 fillType

### ISSUE THREAD

**Issue by BennyT93** [stijnwop/manureSystem#184]
Hi Guys,

I was chatting to jopi98 in your discord on this and they suggested raising a ticket here as a potential future enhancement.

I've setup a slurry buy point to support the manure system mod. Without the mod active you can select whether you want to purchase Digestate or Slurry whereas with the manure system active, you can only purchase digestate. I've added the fillType liquidManure to the connector entry in the xml to make it purchase slurry instead of digestate but would it be possible to still allow the player to select which fillType they would like to purchase from the buying station?

Awesome mod, so glad to see it make its way back for FS22. Thanks in advance.

Regards,
GlengarBoy

**Comment by Tubezki** [comment]
Yes, this is an issue, it should definitely be possible to let player choose output when it is a buying station.
